Foundation/NSHTTPCookieStorage.swift

```diff
@@ -47,7 +46,13 @@ open class HTTPCookieStorage: NSObject {
         allCookies = [:]
         cookieAcceptPolicy = .always
         super.init()
-        cookieFilePath = filePath(path: _CFXDGCreateDataHomePath()._swiftObject, fileName: "/.cookies." + cookieStorageName)
+        let bundlePath = Bundle.main.bundlePath
+        var bundleName = bundlePath.components(separatedBy: "/").last!
```

Comment:
@pushkarnk Yes, 'bundlePath'  is always a non-nil String that will ensure that the value returned by 'bundlePath.components(separatedBy: "/")' can be safely unwrapped.